src/dlr_tvm.cc

```diff
@@ -31,6 +31,8 @@ ModelPath dlr::GetTvmPaths(std::vector<std::string> dirname) {
       paths.params = filename;
     } else if (filename == "version.json") {
```

Comment:
BTW this comparison doesn't work in C++ I think. Left it as is for now. I should probably fix it.

Reply:
in Java it does not work. In C++ it works. http://www.cplusplus.com/reference/string/string/operators/